docs/security-champion/2-new_security_champion.md

```diff
@@ -0,0 +1,8 @@
+
+# New Security Champion 👶
+
+✅ Introduce yourself in our [Slack-channel](https://equinor.slack.com/archives/C036HGPBJ04) 👋  
+✅ Ensure all your code is scanned by a SAST (ex. by Snyk)  
+✅ Have a discussion in your team about [security requirements](https://equinor.github.io/appsec/resources/security_requirements/), and get a common understanding of what can't go wrong (and maybe write some requirements down)  
+✅ Facilitate a [threat modelling session](https://equinor.github.io/appsec/resources/threat_modelling/), looking at the high level architecture of your project  
```

Comment:
Add relative links, then it will work in markdown as well, `e.g ../../resources/threat_modelling.md`
Mkdocs will create the correct URL's during bundling